README.md

```diff
@@ -305,7 +287,8 @@ If you have found HIVE useful for your research, please cite our [technical repo
 
 ## Contributors
 
-HIVE is currently maintained by Nick Reinicke ([@nreinicke] (https://github.com/nreinicke)) and Rob Fitzgerald ([@robfitzgerald](https://github.com/robfitzgerald)). It would not be what it is today without the support of: 
+HIVE is currently maintained by Nick Reinicke ([@nreinicke] (<https://github.com/nreinicke>)) and Rob Fitzgerald ([@robfitzgerald](https://github.com/robfitzgerald)). It would not be what it is today without the support of:
```

Comment:
there's a space in the hyperlink for your github account so as a result it doesn't parse correctly:

Nick Reinicke ([@nreinicke] (https://github.com/nreinicke)...

should be: Nick Reinicke ([@nreinicke](https://github.com/nreinicke)...